### DISCUSSION TITLE
search result  list parent and nested groups

### DISCUSSION THREAD

**Discussion by atvcaptain** [keepassxreboot/keepassxc#7378] [Ideas]
add a option to list the parent groups too,

i have many nested groups with same name see example

Group1
          - Admins
Group2
          -Admins

search found all entry's but list only the nested group 
i can't recognize from which parent group the the entry comes from

please add this to the next version thx

**Comment by droidmonkey** [comment]
You can add a group filter to your search: `g:"Group1"`
